docs/background/velocities.rst

```diff
@@ -0,0 +1,521 @@
+
+Here, :math:`s` is the fractional pulsar--screen distance
+(with :math:`0 \leq s \leq 1`), given by
+
+.. math::
+
+    s = 1 - \frac{ d_\mathrm{s} }{ d_\mathrm{p} }.
+
+.. image:: images/sketch-distances.svg
+    :align: center
+    :alt: sketch with distances
+
+
+The effective velocity
+======================
+
+The scattering of the pulsar's radiation by the screen gives rise to an
+interference pattern of bright and dark fringes in space. Because the pulsar,
+the screen, and Earth all move with respect to one another, an Earth-based
+observation will probe different points in this pattern over time.
+
+[add figure showing interference pattern in observing plane]
+
+The sky-plane velocity (i.e., the velocity component that is perpendicular
+to the direct line of sight) of the interference pattern relative to Earth,
+known as the effective velocity, is given by
+
+.. math::
+
+    \vec{v}_\mathrm{eff} = \frac{ 1 }{ s } \vec{v}_\mathrm{lens,sky}
+        - \frac{ 1 - s }{ s } \vec{v}_\mathrm{p,sky}
+        - \vec{v}_\mathrm{\oplus,sky}.
+
+Here, :math:`\vec{v}_\mathrm{p,sky}`, :math:`\vec{v}_\mathrm{lens,sky}`, and
+:math:`\vec{v}_\mathrm{\oplus,sky}` are the sky-plane velocities of the pulsar,
+the screen, and Earth, respectively, all specified relative to the Solar
+System's barycentre.
+The pulsar's sky-plane velocity can be split into a systemic component
+:math:`\vec{v}_\mathrm{p,sys,sky}` (corresponding to the system's proper
+motion), specified relative to the Solar System's barycentre, and an orbital
+component :math:`\vec{v}_\mathrm{p,orb,sky}`, specified relative to the
+pulsar system's barycentre: :math:`\vec{v}_\mathrm{p,sky} =
+\vec{v}_\mathrm{p,sys,sky} + \vec{v}_\mathrm{p,orb,sky}`.
+
+[add figure explaining the 1/s, -(1-s)/s, and -1 factors]
+
+Since we consider a one-dimensional lens, only velocity components parallel to
+the line of lensed images (marked by subscript ':math:`\parallel`' below) have
+any effect on the shift of the interference pattern. Thus, the equation for the
+effective velocity, considering only the relevant velocity components, becomes
+
+.. math::
+
+    v_\mathrm{eff,\parallel} = \frac{ 1 }{ s } v_\mathrm{lens,\parallel}
+        - \frac{ 1 - s }{ s } v_\mathrm{p,\parallel} - v_{\oplus,\parallel}.
+    \label{eq_v_eff} \tag{1}
+
+Again, the pulsar's term can be split into a systemic and an orbital component:
+:math:`v_\mathrm{p,\parallel} = v_\mathrm{p,sys,\parallel} +
+v_\mathrm{p,orb,\parallel}`.
+
+In the following, we examine how the different velocities in equation
+:math:`\ref{eq_v_eff}` that contribute to :math:`v_\mathrm{eff,\parallel}`
+depend on the physical parameters of the pulsar system and the scattering
+screen. The first of these velocities, :math:`v_\mathrm{lens,\parallel}`,
+is a free parameter in the problem, leaving us to examine
+:math:`v_\mathrm{p,sys,\parallel}`, :math:`v_\mathrm{p,orb,\parallel}` and
+:math:`v_{\oplus,\parallel}`.
+
+
```

Comment:
Might it make sense to also give the expression in terms of proper motions? I find it rather insightful that then all the s-terms drop (with mu_earth=v_earth/d_eff)

Reply:
Added.

Reply:
???

Reply:
Oops, sorry, it clearly is added here, but I was not seeing it in my rendering...